### PR TITLE
[DOCS] Add anomaly detection example for geographic data

### PR DESCRIPTION
**Pull request by lcawl**
Related to https://github.com/elastic/kibana/pull/88416 and https://github.com/elastic/kibana/pull/88880

This PR adds an example of how to create anomaly detection jobs that use the lat_long function and how to view that type of data in the Data Visualizer and the Anomaly Explorer.

### Preview

https://stack-docs_1631.docs-preview.app.elstc.co/guide/en/machine-learning/master/geographic-anomalies.html

### CODE REVIEW

**Review by szabosteve — COMMENTED** [review]
Thanks for adding this example to the pool. Left a couple of suggestions to fix the section IDs.
The [_Anomaly detection examples_](https://www.elastic.co/guide/en/machine-learning/master/anomaly-examples.html) page aggregates all the examples in a bulleted list. Could you please add this page to that list as well?

File: docs/en/stack/ml/anomaly-detection/geographic-anomalies.asciidoc [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+{kib}.
+
+TIP: If you used APIs to create the jobs and {dfeeds}, you cannot see them
+in {kib} until you follow the prompts to synchronize the necessary saved objects.
+
+When you select a period that contains an anomaly in the swim lane results, you
+can see a map of the typical and actual coordinates. For example, the `jackson`
+user ID typically shops in Los Angeles so their purchase in New York is
+anomalous in the eCommerce sample data:
+//TBD: Is it working as designed that the map only appears after you click the swim lane?
```
Comment (alvarezmelissa87) [inline docs/en/stack/ml/anomaly-detection/geographic-anomalies.asciidoc:215]: Should this be removed?
Also, the answer is yes (for data with geo point data) - it works as a type of chart - like the ones that appear already above the anomaly table once you click a swimlane.

Comment (lcawl) [reply docs/en/stack/ml/anomaly-detection/geographic-anomalies.asciidoc]: Thanks!  I've removed that comment.

**Review by alvarezmelissa87 — APPROVED** [review]
LGTM ⚡

**Pull request merged**